src/tests/unittests/WireTests.cpp

```diff
@@ -436,6 +441,30 @@ TEST_F(WireTests, StructureOfObjectArrayArgument) {
     FlushClient();
 }
 
+// Test that the wire is able to send structures that contain objects
+TEST_F(WireTests, StructureOfStructureArrayArgument) {
+    static constexpr int NUM_BINDINGS = 3;
+    nxtBindGroupBinding bindings[NUM_BINDINGS] {
+      { 0, NXT_SHADER_STAGE_BIT_VERTEX, NXT_BINDING_TYPE_SAMPLER, 2 },
+      { 1, NXT_SHADER_STAGE_BIT_VERTEX, NXT_BINDING_TYPE_SAMPLED_TEXTURE, 1 },
+      { 2, static_cast<nxtShaderStageBit>(NXT_SHADER_STAGE_BIT_VERTEX | NXT_SHADER_STAGE_BIT_FRAGMENT), NXT_BINDING_TYPE_UNIFORM_BUFFER, 1 },
```

Comment:
the static_cast here is unfortunate, but not sure how to avoid it. (causes a type error :int where there should be an nxtShaderStageBit).